src/pages/home/report/ReportActionsList.js

```diff
@@ -137,15 +139,22 @@ class ReportActionsList extends React.Component {
         const shouldDisplayNewIndicator = this.props.report.newMarkerSequenceNumber > 0
             && item.action.sequenceNumber === this.props.report.newMarkerSequenceNumber;
         return (
-            <ReportActionItem
-                reportID={this.props.report.reportID}
-                action={item.action}
-                displayAsGroup={ReportActionsUtils.isConsecutiveActionMadeByPreviousActor(this.props.sortedReportActions, index)}
-                shouldDisplayNewIndicator={shouldDisplayNewIndicator}
-                isMostRecentIOUReportAction={item.action.sequenceNumber === this.props.mostRecentIOUReportSequenceNumber}
-                hasOutstandingIOU={this.props.report.hasOutstandingIOU}
-                index={index}
-            />
+            <OfflineWithFeedback
+                onClose={() => ReportActions.deleteClientAction(this.props.report.reportID, item.action.clientID)}
+                pendingAction={item.action.pendingAction}
+                errors={item.action.errors}
```

Comment:
What if `pendingAction` or `errors` doesn't exist on the `action`. I think `lodashGet` should be used here.

Reply:
if `pendingAction` or `errors` doesn't exist, but `item.action` does exist then they will just be `undefined` and we are probably OK because of this:

https://github.com/Expensify/App/blob/f5f076a13bb4a991dbeaf32a8f7a7b0625e8a3cc/src/components/OfflineWithFeedback.js#L49-L51

If the action itself is `undefined` then the JS would throw a `TypeError`, but it doesn't seem possible to me.